### ISSUE TITLE
Network Architecture is wrong

### ISSUE THREAD

**Issue by kevinchristensen1** [zhixuhao/unet-multi#4]
I was able to solve this issue by changing the Reshape args for last layers of the model.
Instead of the current model definition in unet.py (notice the input_shape arg for the Keras Reshape):
```
  reshape = Reshape((self.img_rows * self.img_cols, 12), input_shape=(self.img_rows, self.img_cols, 12))(conv9) 
print("reshape shape:", reshape.shape)  

# permute = Permute((2, 1))(reshape) 
# print("permute shape:", permute.shape)  

activation = Activation('softmax')(reshape) 

```
After 10 epochs I got this result:
![image](https://user-images.githubusercontent.com/38863271/44629617-a8dce600-a95a-11e8-8c12-a87c9de21b98.png)
![image](https://user-images.githubusercontent.com/38863271/44629613-8f3b9e80-a95a-11e8-8c5e-0b798a1b71c2.png)

_Originally posted by @shabtayor in https://github.com/zhixuhao/unet-multi/issues/1#issuecomment-416045794_

**Comment by shangmou** [comment]
> I was able to solve this issue by changing the Reshape args for last layers of the model.
> Instead of the current model definition in unet.py (notice the input_shape arg for the Keras Reshape):
>   reshape = Reshape((self.img_rows * self.img_cols, 12), input_shape=(self.img_rows, self.img_cols, 12))(conv9) 
> print("reshape shape:", reshape.shape)  
> 
> # permute = Permute((2, 1))(reshape) 
> # print("permute shape:", permute.shape)  
> 
> activation = Activation('softmax')(reshape) 
> 
> 
> After 10 epochs I got this result:
> 
> 
> Originally posted by @shabtayor in #1 (comment)

Hello, may I ask if this set of code can be provided ?
